.github/workflows/check.yml

```diff
@@ -1,53 +1,82 @@
-name: check
+name: Check
 
-on: [pull_request, push]
+on:
+  push:
+  pull_request:
+    branches:
+      - master
 
 jobs:
+  check_duplicate_runs:
+    name: Check for duplicate runs
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          concurrent_skipping: always
+          cancel_others: true
+          skip_after_successful_duplicate: true
+          paths_ignore: '["**/README.md", "**/CHANGELOG.md", "**/LICENSE.md"]'
+          do_not_skip: '["pull_request"]'
+
   mix_check:
     name: mix check (Elixir ${{matrix.elixir}}, OTP ${{matrix.otp}})
+    runs-on: ubuntu-18.04
+    needs: check_duplicate_runs
+    if: ${{ needs.check_duplicate_runs.outputs.should_skip != 'true' }}
 
     strategy:
       matrix:
-        include:
-          - elixir: '1.10.4'
-            otp: '23.0.2'
-          - elixir: '1.7.0'
-            otp: '19.0'
-
-    runs-on: ubuntu-18.04
+        elixir:
+        - "1.7"
+        - "1.8"
+        - "1.9"
+        - "1.10"
+        - "1.11"
+        otp:
+        - "20.0"
```

Comment:
Previous solution tested OTP as low as 19 (lowest for lowest supported Elixir). Do you think it's safe to ignore it? If it ever came as a default install for Elixir builds on popular platforms like Ubuntu or Mac, then it may not be the best idea.

Reply:
Maybe it is not safe to ignore it, but now it is not possible to install this OTP version on Github Actions ubuntu machine, neither 20.04 nor 18.04.
That's why I removed it from the matrix. I see that you done just the same in the master branch.

Reply:
Yup, I've just found the same. More info: https://github.com/erlef/setup-elixir/issues/7.

Worked on Travis just fine 😛